src/test/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ChipsServiceTest.java

```diff
@@ -41,18 +45,30 @@
 
     @Test
     void testSendingToChipsCreatesCorrectRequest() throws ServiceException {
+        ReflectionTestUtils.setField(chipsService, "attachmentDownloadUrlPrefix", DOWNLOAD_URL_PREFIX);
         Objection objection = Utils.getTestObjection(
                 OBJECTION_ID, REASON, COMPANY_NUMBER, USER_ID, EMAIL, LOCAL_DATE_TIME,
                 Utils.buildTestObjectionCreate("Joe Bloggs", false));
+        List<Attachment> attachments = new ArrayList<>();
+        Utils.setTestAttachmentsWithLinks(attachments);
+        objection.setAttachments(attachments);
 
-        chipsService.sendObjection(REQUEST_ID, objection);
+        chipsService.sendObjection(OBJECTION_ID, objection);
```

Comment:
the first param should be the http request id used for logging, so it looks like REQUEST_ID was correct.

Reply:
Are you sure?

Reply:
There is no JSON Property objectioId?

Reply:
Alright changed these back , not sure whwere that leaves the chipsRequest class as the first parameter is for an objection ID, i.e do we have two ids, or one or the other?

Reply:
Hold on ignore that.

Reply:
Made sure request and objection ids ar ein the right places.

Reply:
have a look at https://github.com/companieshouse/strike-off-objections-api/blob/375c15a50c3cc7ed21522ea88da7377d74e39bbf/src/main/java/uk/gov/companieshouse/api/strikeoffobjections/service/impl/ChipsService.java#L22
you can see objectionId is obtained from Objection so we don't need to pass it separately.

in that method, request id is passed into https://github.com/companieshouse/strike-off-objections-api/blob/fc87dec5966da6014e294f5dd9e70768c94a1526/src/main/java/uk/gov/companieshouse/api/strikeoffobjections/chips/ChipsClient.java#L34
You can see in there the request id is used in our logging, we've been using the http request id in all our logging so far so we can trace the request through the logs

Reply:
I know just released.